config/initializers/postgres_required_versions.rb

```diff
@@ -5,13 +5,13 @@ def initialize(*args)
   end
 
   def check_version
-    msg = "The version of PostgreSQL being connected to is incompatible with #{Vmdb::Appliance.PRODUCT_NAME} (10 required)"
+    msg = "The version of PostgreSQL being connected to is incompatible with #{Vmdb::Appliance.PRODUCT_NAME} (13 required)"
 
-    if postgresql_version < 90500
+    if postgresql_version < 100000
       raise msg
     end
 
-    if postgresql_version < 100000 || postgresql_version >= 130000
+    if postgresql_version < 130000 || postgresql_version >= 140000
```

Comment:
Do we want to lock this down to only allow PG 13 in production mode for now or maybe just allow it now and lock it down later?

Reply:
We're now very lenient and allow 10-14, warn on 14+ and disallow 9 and lower.